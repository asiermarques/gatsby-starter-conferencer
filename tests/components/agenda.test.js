import React from "react"
import renderer from "react-test-renderer"
import Agenda from "../../src/components/agenda"
import AgendaConfigFixture from "../__fixtures/AgendaConfig"
import SpeakerFixture from "../__fixtures/Speaker"

describe("Agenda", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Agenda agenda={AgendaConfigFixture} speakers={[SpeakerFixture]}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})