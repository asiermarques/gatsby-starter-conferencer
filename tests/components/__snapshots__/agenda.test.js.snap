// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Agenda renders correctly 1`] = `
<section
  id="agenda"
>
  <div
    className="container"
  >
    <h2
      className="text-center"
    >
      Agenda
    </h2>
    <div
      className="table-responsive-sm"
    >
      <table
        className="table table-striped stacktable large-only"
        id="agenda-table"
      >
        <thead>
          <tr>
            <th />
            <th
              width="40%"
            >
              Olympia Room
            </th>
            <th
              width="40%"
            >
              Niza
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              class="time"
            >
              10:00-11:00
            </td>
            <td>
              <a
                href="sara-smith#talk"
              >
                <h5>
                  Talk title demo
                </h5>
                <p>
                  Sara Smith
                </p>
              </a>
            </td>
            <td>
              <a
                href="sara-smith#talk"
              >
                <h5>
                  Talk title demo
                </h5>
                <p>
                  Sara Smith
                </p>
              </a>
            </td>
          </tr>
          <tr>
            <td
              class="time"
            >
              11:00-12:00
            </td>
            <td>
              <p>
                test
              </p>
            </td>
            <td>
              <p>
                test
              </p>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</section>
`;
